mu-plugins/10up-plugin/includes/core.php

```diff
@@ -108,12 +108,14 @@ function get_enqueue_contexts() {
  * @param string $script Script file name (no .js extension)
  * @param string $context Context for the script ('admin', 'frontend', or 'shared')
  *
- * @return string|WP_Error URL
+ * @throws \RuntimeException If an invalid $context is specified.
+ *
+ * @return string URL
  */
 function script_url( $script, $context ) {
 
 	if ( ! in_array( $context, get_enqueue_contexts(), true ) ) {
-		return new WP_Error( 'invalid_enqueue_context', 'Invalid $context specified in TenUpPlugin script loader.' );
+		throw new \RuntimeException( 'Invalid $context specified in TenUpPlugin script loader.' );
```

Comment:
How do these get reported in WP? Anything to be aware of relating to this change?